src/v2/view-builder/utils/LinksUtil.js

```diff
@@ -80,12 +80,12 @@ const goBackLink = (appState) => {
   return [];
 };
 
-const getSkipSetupLink = (appState) => {
+const getSkipSetupLink = (appState, linkName) => {
   if (appState.hasRemediationObject(RemediationForms.SKIP)) {
     return [
       {
         'type': 'link',
-        'label': loc('oie.enroll.skip.setup', 'login'),
+        'label': linkName ? linkName : loc('oie.enroll.skip.setup', 'login'),
```

Comment:
Consider using the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) here, one of the newer [enhancements included](https://babeljs.io/docs/en/babel-plugin-proposal-nullish-coalescing-operator) with our Babel present.

```suggestion
        'label': linkName ?? loc('oie.enroll.skip.setup', 'login'),
```